ksql-engine/src/test/java/io/confluent/ksql/planner/plan/JoinNodeTest.java

```diff
@@ -367,30 +384,11 @@ public void shouldThrowOnPartitionMismatch() {
     );
   }
 
-  @Test
-  public void shouldHaveAllFieldsFromJoinedInputs() {
```

Comment:
Any specific reason for removing this test?

Reply:
It's a duplicate of `shouldHaveFullyQualifiedJoinSchema`.